example/20-componentconfig-gardenlet.yaml

```diff
@@ -192,6 +192,7 @@ etcdConfig:
 #   reelectionPeriod: 5s
 #   etcdConnectionTimeout: 5s
 # monitoring:
+#   enabled: true
```

Comment:
Since this is only used to disable the monitoring for shoot and will not affect the monitoring stack for seeds that is deployed in the `garden` namespace I would also vote to make the option: `monitoring.shoot.enabled`